Shared/Samples/Download preplanned map area/DownloadPreplannedMapAreaView.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct DownloadPreplannedMapAreaView: View {
+    /// A Boolean value indicating whether the sample info view is visible.
+    @Environment(\.isSampleInfoViewVisible) private var isSampleInfoViewVisible
```

Comment:
Is this needed?
```suggestion
```

Reply:
no, it is a remnant of a previous way that the branch this PR came from was cleaning things up.